### DISCUSSION TITLE
FEEDBACK: Vote on Next Features

### DISCUSSION THREAD

**Discussion by offizium-berndstorath** [wg-easy/wg-easy#1307] [General]
Vote what Features you would like to be implemented next.
React to the Comment you like most. Can be multiple (just don't vote on all).
Not a guarantee the features will be implemented/accepted.
There is no deadline, its just to gather some of your feedback and thoughts

**Comment by offizium-berndstorath** [comment]
### Shutdown/Restart wireguard
Integrate a button for straightforward shutdown and/or restart of the VPN server.

**Comment by offizium-berndstorath** [comment]
### User management & groups
be able to create multiple users and assign them clients
Implement user grouping to manage permissions, such as enabling or disabling internet access for specific groups.

**Reply by offizium-berndstorath** [reply]
being able to group by person would also be helpful (https://github.com/wg-easy/wg-easy/discussions/1310)

**Comment by offizium-berndstorath** [comment]
### UI Onboarding & Password managment
Dont set password using environment variables. but entirely through the ui.
Provide a seamless interface for users to update their passwords at their convenience.

**Comment by offizium-berndstorath** [comment]
### Firewall interface
Provide a seamless interface for users to manage rules.

**Comment by offizium-berndstorath** [comment]
### Integrations
be able to set e.g. email for a client and send the config to that email through the ui

**Comment by offizium-berndstorath** [comment]
### Multiple Instance support
be able to configure clients on multiple distinct wg-easy instances (e.g. for geoblocking)

**Comment by offizium-berndstorath** [comment]
### Configure Client through UI
be able to configure Allowed IPs, Keepalive, DNS per Client

**Comment by offizium-berndstorath** [comment]
### Alerts
Send an alert (e.g. on connect or disconnect) using Discord, Telegram, etc.

**Comment by offizium-berndstorath** [comment]
### Persistent Traffic Stats  & Quota Limits
Persist how much Traffic a Client used even after disable,enable and reboot. This also allows to implement Quota Limits

**Reply by djparus** [reply]
Yes, there really are not enough statistics and traffic quotas for each client.
I'm really looking forward to this feature

**Comment by offizium-berndstorath** [comment]
### IPv6
Be able to use IPv6 only servers and access IPv6 only websites

**Comment by offizium-berndstorath** [comment]
### CIDR
Implement proper CIDR Support (10.0.0.0/24 instead of 10.0.0.x)

**Comment by offizium-berndstorath** [comment]
### Documentation & Website
Add better Documentation, better Docker workflow and/or a Website with Instructions and examples
